packages/@adobe/spectrum-css-temp/components/steplist/index.css

```diff
@@ -218,7 +218,8 @@ governing permissions and limitations under the License.
 
     padding-inline-start: calc(var(--spectrum-steplist-chevron-gap) - var(--spectrum-steplist-marker-label-gap));
     &::before {
-      content: 'x';
+      content: "*";
+      content: "*" / "";
```

Comment:
fyi safari doesn't support this syntax, and neither does firefox. safari supports a non-standard `alt` property instead. see what I did in the Menu RAC docs for that. But neither are supported in Firefox. The best solution might be to not use a `:before`... 😞